doc/classes/bool.xml

```diff
@@ -6,24 +6,50 @@
 	<description>
 		Boolean is a built-in type. There are two boolean values: [code]true[/code] and [code]false[/code]. You can think of it as an switch with on or off (1 or 0) setting. Booleans are used in programming for logic in condition statements, like [code]if[/code] statements.
 		Booleans can be directly used in [code]if[/code] statements. The code below demonstrates this on the [code]if can_shoot:[/code] line. You don't need to use [code]== true[/code], you only need [code]if can_shoot:[/code]. Similarly, use [code]if not can_shoot:[/code] rather than [code]== false[/code].
-		[codeblock]
+		[codeblocks]
+		[gdscript]
 		var can_shoot = true
 
 		func shoot():
 		    if can_shoot:
 		        pass # Perform shooting actions here.
-		[/codeblock]
+		[/gdscript]
+		[csharp]
+		private bool _canShoot = true;
```

Comment:
If this is supposed to be private, you should change the GDScript code example too so it's consistent (`var _can_shoot = true`).

Reply:
fixed all occurrences.